zetaclient/chains/solana/signer/signer.go

```diff
@@ -219,6 +240,56 @@ func (signer *Signer) prepareWithdrawTx(
 	return tx, nil
 }
 
+func (signer *Signer) prepareWithdrawSPLTx(
+	ctx context.Context,
+	cctx *types.CrossChainTx,
+	height uint64,
+	logger zerolog.Logger,
+) (*solana.Transaction, error) {
+	params := cctx.GetCurrentOutboundParam()
```

Comment:
We could use some defensive approach and check cctx is nil before using it. Same for params as it's a pointer.